Tests/TestUtilities/SQPropertiesFileReader.cs

```diff
@@ -1,30 +1,42 @@
 ﻿//-----------------------------------------------------------------------
-// <copyright file="FilePropertiesProvider.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="SQPropertiesFileReader.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
Renamed the file and moved it to TestUtilties. The old name ("FilePropertiesProvider") was confusingly similar to "AnalysisPropertyFileProvider".